test/torchaudio_unittest/transforms/torchscript_consistency_impl.py

```diff
@@ -18,6 +18,9 @@ def _assert_consistency(self, transform, tensor):
         tensor = tensor.to(device=self.device, dtype=self.dtype)
         transform = transform.to(device=self.device, dtype=self.dtype)
 
+        # Perform dry-run so that UninitializedBuffer/Parameter are materialized
```

Comment:
I don't think this is necessary if taking the `dtype` initialization approach

Reply:
oops I forgot to remove it.